tracer/test/test-applications/integrations/Samples.WebRequest/Program.cs

```diff
@@ -44,8 +47,6 @@ public static async Task Main(string[] args)
                 Console.WriteLine();
                 Console.WriteLine("Stopping HTTP listener.");
             }
-
-            await Tracer.Instance.ForceFlushAsync();
```

Comment:
Dropping this doesn't change anything?

Reply:
Hmm, perhaps it might introduce test flakiness by removing this. I've reinserted a reflection-based call to ForceFlushAsync in 373eb6aa97c5faae92d91769ce3ccdd7ecb8c021